src/docs/respectful-code.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Respectful code"
+description: "Inclusivity is central to V8’s culture, and our values include treating each other with dignity. As such, it’s important that everyone can contribute without facing the harmful effects of bias and discrimination."
+---
+
+Inclusivity is central to V8's culture, and our values include treating each other with dignity. As such, it’s important that everyone can contribute without facing the harmful effects of bias and discrimination. However, terms in our codebase, UIs, and documentation can perpetuate that discrimination. This document sets forth guidance which aims to address disrespectful terminology in code and documentation.
+
+## Policy
+
+Terminology that is derogatory, hurtful, or perpetuates discrimination, either directly or indirectly, should be avoided.
+
+## What is in scope for this policy?
```

Comment:
Maybe append `{ #scope }` here for nice IDs? Same for the other long headings below.

Reply:
```suggestion
## What is in scope for this policy? { #scope }
```